examples/pipelines/rag-cc-pipeline/pipeline.py

```diff
@@ -40,10 +42,8 @@
 embed_text_op = ComponentOp.from_registry(
     name="embed_text",
     arguments={
-        "model_provider": "vertexai",
-        "auth_kwargs": {
-            "project": "<project-id>",
-        },
+        "model_provider": "huggingface",
+        "model": "all-MiniLM-L6-v2",
     },
```

Comment:
I removed the GPU from here since it takes quite a long time to deploy for a demo. Takes around 5 mins to embed the whole dataset with CPU